src/b3d/chisight/gen3d/pixel_kernels/pixel_color_kernels.py

```diff
@@ -73,6 +75,50 @@ def logpdf_per_channel(
         raise NotImplementedError
 
 
+@Pytree.dataclass
+class RenormalizedGaussianPixelColorDistribution(PixelColorDistribution):
+    """
+    Sample a color from a renormalized Gaussian distribution centered around the given
+    latent_color (rgb value), given the color_scale (stddev of the Gaussian).
+
+    The support of the distribution is ([0, 1]^3).
+    """
+
+    def sample(self, key, latent_color, color_scale, *args, **kwargs):
+        return jax.vmap(
+            genjax.truncated_normal.sample, in_axes=(0, 0, None, None, None)
+        )(split(key, latent_color.shape[0]), latent_color, color_scale, 0.0, 1.0)
```

Comment:
How to you feel about using`COLOR_MIN_VAL` and `COLOR_MAX_VAL` instead of hard-coding the magic constant?

Reply:
Will do!